lib/src/extensions.dart

```diff
@@ -1,4 +1,4 @@
-extension on int {
+extension IntTimeExtension on int {
```

Comment:
Giving each Extension a name was the key to making tests work

Reply:
Ah! I knew I was missing something. Thanks!

Reply:
Yah, not sure if that's a bug in the current extensions implementation or by design. Either way, doesn't hurt to give em a name :)

Reply:
Sure. It does make sense